_categories/sensors.md

```diff
@@ -2,18 +2,18 @@
 # Contribution guidelines:
 # https://github.com/osrf/gz-bigindex/blob/master/CONTRIBUTING.md 
 
-title: "Sensors: the sensor catalogue"
+title: "Sensors: The sensor catalogue"
 desc: "How to add already implemented sensors (camera, lasers, IMU, ...) to the gazebo models."
 subcategories: 
   - title: "Altimeter Gazebo sensor"
     items: 
-      - title: 'Basic shapes in Gazebo simulator'
-        url: 'http://packages.osrfoundation.org/gz_big_index/4_sensors.html' # check
-        desc: 'How to insert and use an altimiter simulated sensor in Gazebo'
-        star: true
+      # - title: 'Basic shapes in Gazebo simulator'
+      #   url: 'http://packages.osrfoundation.org/gz_big_index/4_sensors.html' # check
```

Comment:
I think we can link this tutorial here: http://gazebosim.org/tutorials?tut=build_world&cat=build_world

Reply:
This would be more precise -
http://gazebosim.org/tutorials?tut=build_world&cat=build_world#AddingSimpleShapes

Reply:
Btw, should this item ('Basic shapes in Gazebo simulator') be there in the Sensors catalogue?
Wouldn't it be better in one of the Models category? (like ```models_import.md```)

Reply:
You are totally right on both, the tutorial and the category. Please change it.